mempoolspace/client.go

```diff
@@ -0,0 +1,84 @@
+package mempoolspace
+
+import (
+	"encoding/binary"
+	"encoding/hex"
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+)
+
+type Client struct {
+	// meempool parameters
+	uri string // need to implement
+}
+type Utxo struct {
+	Value       btcutil.Amount
+	BlockHeight int32
+	wire.OutPoint
+}
+type respUtxo struct {
+	txid   []byte
+	vout   int
+	status status
+	value  btcutil.Amount
+}
+type status struct {
+	confirmed    bool
+	block_height int32
+	block_hash   []byte
+	block_time   uint64
+}
+type RecommendedFeesResponse struct {
+	minimumFee uint64
+}
+
+func RecommendedFee() (uint64, error) {
```

Comment:
These functions should belong to the Client struct, eg: 
`func (c *Client) RecommendedFee() (uint64, error)`

And then the user should use it like this:

```
c := &Client{baseUrl: "https://mempool.space"}
c.RecommendedFees()
```